tools/RunTests.ps1

```diff
@@ -0,0 +1,194 @@
+<#
+.SYNOPSIS
```

Comment:
What's wrong with `TestAll.ps1` (in root of the repository)?

If that lacks functionality (e.g. run explicit subset of tests) why not improve `TestAll`?

Or perhaps fork it to `Test.cmd` and `tools\Test.ps1` and change `\TestAll.cmd` to run `Test.cmd ...options for existing TestAll.cmd behavior...`